app/views/pregnancies/_new_pregnancy.html.erb

```diff
@@ -2,27 +2,35 @@
 <%= bootstrap_form_for pregnancy do |f| %>
 
   <%= f.fields_for patient do |p| %>
-    <div class="col-md-2">
-      <%= p.text_field :primary_phone, label: "Phone Number" %>
+
+
+<div class="col-md-3">
+    <div class="col-md-6">
+      <%= p.text_field :primary_phone, label: "Number" %>
```

Comment:
I think in the CM context, 'Number' can mean a couple of different things (in particular, a patient code we use internally). Any chance we could change this label to `Phone`?